README.md

```diff
@@ -7,290 +7,505 @@
 
-1. If you use Guzzle (or some other dependency does), you will need to update it to work with Guzzle 6.1.
+When contributing to this SDK, please:
 
-### from 0.6.6
+- Maintain the minimum PHP version (found under `require.php` in `composer.json`)
+- Code to the [PSR-2 standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)
+- Write tests and run them with `composer test`
+- Keep PRs focused and change the minimum number of lines to achieve your goal
 
-1. First is important to read the [API v2 changes document](https://auth0.com/docs/apiv2Changes) to catch up the latest changes to the API.
-2. Update your composer.json file.
- - change the version "auth0/auth0-php": "~1.0"
- - add the new dependency "firebase/php-jwt" : "dev-master"
-3. Now the SDK is PSR-4 compliant so you will need to change the namespaces (sorry **:(** ) to `\Auth0\SDK`
-4. The method `getUserInfo` is deprecated and candidate to be removed on the next release. User `getUser` instead. `getUser` returns an User object compliant with API v2 which is a `stdClass` (check the schema [here](https://auth0.com/docs/apiv2#!/users/get_users_by_id))
+To run tests on the SDK, you'll need to create a `.env` file in the root of this package with the following entries:
 
-## Develop
+- `DOMAIN` - Auth0 domain for your test tenant
+- `APP_CLIENT_ID` - Client ID for the test Application
+- `APP_CLIENT_SECRET` - Client Secret for the test Application
+- `GLOBAL_CLIENT_ID` - Client ID for your tenant (found in Tenant > Settings > Advanced)
+- `GLOBAL_CLIENT_SECRET` - Client Secret for your tenant (found in Tenant > Settings > Advanced)
 
-### _.env_ format
+We're working on test coverage and quality but please note that newer tenants might see errors (typically `404`) for endpoints that are no longer available. Another common error is a `429` for too many requests. 
 
-- GLOBAL_CLIENT_ID
-- GLOBAL_CLIENT_SECRET
-- DOMAIN
+## Troubleshooting
 
-### Install dependencies
-This SDK uses [Composer](http://getcomposer.org/doc/01-basic-usage.md) to manage its dependencies.
+> I am getting `curl error 60: SSL certificate problem: self signed certificate in certificate chain` on Windows
 
-## Configure example
+This is a common issue with latest PHP versions under windows (related to a incompatibility between windows and openssl CAs database).
 
-1. Install dependencies
-2. Start your web server on `examples/{example-name}` folder.
-3. Create an OpenID Connect Application on Auth0.
-4. Configure the callback url to point `{your-base-url}\callback.php`.
-5. Open `examples/{example-name}/config.php` and replace all placeholder parameters.
-6. On your browser, open the Auth0 example project. Make sure `index.php` is being loaded.
+- download this CA database `https://curl.haxx.se/ca/cacert.pem` to `c:/cacert.pem`
+- edit your php.ini and add `openssl.cafile=c:/cacert.pem` (it should point to the file you downloaded)
```

Comment:
Suggested change: Edit your php.ini and add `openssl.cafile=c:/cacert.pem`. (It should point to the file you downloaded.)